anatomy/tracks/practice-exercises.md

```diff
@@ -28,4 +28,303 @@ Practice Exercise metadata is defined in the `exercises.practice` key in the [co
 
 ## Files
 
-TODO: describe files of Practice Exercises
+Each Practice Exercise has its own directory within the track's `exercises/practice` directory. The name of the Practice Exercise directory must match the `slug` property of the Practice Exercise, as defined in the [config.json file](./config-json.md#concept-exercises).
+
+A Practice Exercise has three types of files.
+
+### Documentation files
+
+These files are presented to the student to help explain the exercise.
+
+- `.docs/introduction.md`: introduces the setting and background for the exercise (optional)
+- `.docs/introduction.append.md`: additional introduction text to append after the existing introduction (optional)
+- `.docs/instructions.md`: provide instructions for the exercise (required)
+- `.docs/instructions.append.md`: additional introduction text to append after the existing instructions (optional)
+- `.docs/hints.md`: provide hints to a student to help them get themselves unstuck in the exercise (optional)
+
+### Metadata files
+
+These files are _not_ presented to the student, but used to define metadata of the exercise.
+
+- `.meta/config.json`: contains meta information on the exercise (required)
+- `.meta/design.md`: describe the design of the exercise (optional)
```

Comment:
I added this for consistency with concept exercises and because it can be used to document certain design decisions.